libindy_vdr/src/pool/runner.rs

```diff
@@ -50,7 +52,7 @@ impl PoolRunner {
             )
             .unwrap();
             let mut thread = PoolThread::new(pool, receiver);
-            thread.run();
+            thread.run(cache);
```

Comment:
I think it would make more sense to make cache a property of `PoolThread` and leave `run()` with no arguments?

Reply:
Done